http/_client_connection.pony

```diff
@@ -260,9 +261,11 @@ actor _ClientConnection is HTTPSession
           let request = _unsent.shift()?
           let safereq = request.is_safe()
           // Send all of the request that is possible for now.
-          request._write(true, conn)
+          let wr = recover ref Writer end
```

Comment:
This will allocate a new Writer class on every iteration, perhaps it's better to move this to an `embed` field of the actor, so we can keep reusing it.

Reply:
thanks for the hint. I just did that

Reply:
Great!